WalletWasabi.Tests/UnitTests/TransactionProcessorTests.cs

```diff
@@ -340,4 +525,13 @@ private static OutPoint GetRandomOutPoint()
 			return new OutPoint(RandomUtils.GetUInt256(), 0);
 		}
 	}
+
+	static class TransactionProcessorExtensions
```

Comment:
Is it ok for this class to be declared in the same file?

Reply:
Yes, because it is an extension method used only by the tests on this file.